src/components/clusterConfiguration/HostsTable.tsx

```diff
@@ -67,7 +68,7 @@ const hostToHostTableRow = (openRows: OpenRows) => (host: Host): IRow => {
       // visible row
       isOpen: !!openRows[id],
       cells: [
-        id,
+        inventory.hostname,
```

Comment:
The hostname is not set until the host gets discovered.

```suggestion
        inventory.hostname || EMPTY,
```

Reply:
done